pyproject.toml

```diff
@@ -13,3 +13,6 @@ select = [
     "UP",   # pyupgrade
     "W",    # pycodestyle warnings
 ]
+
+[tool.ruff.lint.pylint]
+max-args = 6
```

Comment:
Increasing the maximum allowed arguments to functions from the default of 5 to 6, to avoid these errors:

```
scripts/datacube.py:293:5: PLR0913 Too many arguments in function definition (6 > 5)
scripts/datacube.py:428:5: PLR0913 Too many arguments in function definition (6 > 5)
```

Alternatively, we could look at simplifying those functions to have less arguments.